src/testcafe/models.ts

```diff
@@ -1,3 +1,17 @@
+export {};
+declare global {
+  interface TestController {
```

Comment:
Please provide link to GH issues why we need to provide our own interfaces for certain TestCafé types/interfaces and why we're not declaring the entire interface.

Reply:
Added the GH issue detailing why the TestController interface is needed and why dexlaring the entire interface is unnessesary.